src/Poster.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react'
 import './Poster.css'
+import { Link } from 'react-router-dom'
 
 const Poster = ({ poster, title, rating, releaseDate, viewMovieInfo, id, formatDate }) => {
   return (
     <article id={id} className="poster">
       <section className="poster-image-container">
         <img className="poster-image" src={poster} alt={title}/>
         <div className="middle">
-          <button className="see-details-btn" onClick={() => viewMovieInfo(id)}>See More Details </button>
+          <Link to={`/${id}`}>
+            <button className='see-details-btn'>See More Details</button>
```

Comment:
Had to put the button inside of the link so that it would display as a button and not highlighted text.